### ISSUE TITLE
Issue Details > Scroll to unread automatically

### ISSUE THREAD

**Issue by Victa** [gitscout/gitscout-feedback#15]
That would be great to scroll to the latest unread message when I open an updated issue from the list.

**Comment by areskub** [comment]
Oh thanks God that's not just me then.

**Comment by SamuAlfageme** [comment]
Yup, while navigating some really long issues, you have to scroll to the end manually. A nice and small button in the right corner would be a useful way to get there. (Or just automatically as suggested by @Victa)

**Comment by cjdowner** [comment]
What about reversing the order of the 'timeline'? Most recent first. In general, it feels borked to scroll up to view the past.

**Comment by areskub** [comment]
The reason (originally) why we put most recent at the bottom was because the input box is at the bottom and we didn't want to have it disconnected from the place where youre comment will appear.

Now since the comment box extends over the timeline and does not mimic the appearance of the soon-to-be message, that might be do-able.

What do you think @Adrien-Griveau @michael-lefebvre @jeremybenaim ?

**Comment by jonyheim** [comment]
Auto-scrolling to the latest unread message (like @Victa suggested) would most likely have the least downsides since you are automatically taken to the content you want most, while maintaining a natural conversation thread. Just my 2¢ – awesome app btw, our office is really excited about it. 🍻
